etl/LISOS_PIMS/02_PSP_PIMS_ETL_OPERATIONAL_SS_PLACEHOLDER_ADDRESS.sql

```diff
@@ -0,0 +1,29 @@
+IF NOT EXISTS (SELECT * FROM [PIMS_ADDRESS] WHERE STREET_ADDRESS_1 = 'LISOS Placeholder Address')
```

Comment:
In order to get around required fields (leases require a moti_person_id, organizations require an address) the etl inserts a placeholder person and address to flag users that data is missing from ETL'd rows.

Reply:
This also allows us to actually etl the data in, since those fields are non-nullable